sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/config/CosmosConfig.java

```diff
@@ -11,22 +11,36 @@
  */
 public class CosmosConfig {
 
+    private final String database;
+
```

Comment:
Yes, `CosmosConfig` - will be confined to single database. 
So for each database, you will create separate `CosmosConfig`. 
This works perfectly for customers who want to have different logging for each database configuration. 
 
Since `CosmosConfig` is just a spring bean, it will not have any performance issues at all. End users can keep separate configurations for separate database.